i18n/nb.po

```diff
@@ -133,8 +136,8 @@ msgstr ""
 
 msgid "{{count}} selected"
 msgid_plural "{{count}} selected"
-msgstr[0] "{{count}} valgt"
-msgstr[1] "{{count}} valgt"
+msgstr[0] ""
```

Comment:
@Philip-Larsen-Donnelly Any idea why these translations (with "count") have been removed in some of the languages?